.github/workflows/tests.yml

```
@@ -11,6 +11,22 @@ on:
       - main
 
 jobs:
+  tox_test:
+    name: Tox test
+    strategy:
+      matrix:
+        tox_env: [py311, py312, py313, py314]
+    # Use GitHub's Linux Docker host
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)